xbmc/guilib/GUISettingsLabelControl.cpp

```diff
@@ -0,0 +1,199 @@
+/*
+ *      Copyright (C) 2005-2015 Team XBMC
+ *      http://xbmc.org
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with XBMC; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "GUISettingsLabelControl.h"
+#include "GUIWindowManager.h"
+#include "GUIDialog.h"
+#include "GUIFontManager.h"
+
+using namespace std;
```

Comment:
Dont't drag in all of std, we use it all over the place but we can avoid old sins in new files :)